scripts/generate_index_pages.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * Generates index.html pages for the home page and all state / county pages,
+ * filling the meta tags for OpenGraph / Twitter with the appropriate title /
+ * description / image information.
+ *
+ * You can run via `yarn generate-index-pages`
+ */
+
+import fs from 'fs-extra';
+import path from 'path';
+import _ from 'lodash';
+import urlJoin from 'url-join';
+import US_STATE_DATASET from '../src/components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
+import ShareImageUrlJSON from '../src/assets/data/share_images_url.json';
+import { STATES } from '../src/common';
+
+// We don't care about the values here, but this is a cheap way to determine all
+// of the counties we have any data for and are therefore share-able.
+import CalculatedCountyInterventionJSON from '../src/assets/data/calculated_county_interventions.json';
+const COUNTIES = Object.keys(CalculatedCountyInterventionJSON);
+
+type MetaTags = {[name: string]: string}
+
+function homePageTags(imageUrl: string): MetaTags {
+  // Most of the tags are set correctly in public/index.html so we don't need to
+  // replace them.
+  return {
+    'og:image:url': imageUrl,
+    'twitter:image': imageUrl,
+  };
+}
+
+function locationPageTags(fullImageUrl: string, canonicalUrl: string, locationName: string): MetaTags {
+  const title = `Is ${locationName} ready to reopen?`;
+  const description = 'Real-time modeling and metrics to understand where we stand against COVID.';
```

Comment:
Note that these are provisional.  I'll work with @schlosser, etc. on final copy.